tests/framework/Optimizers/goldsteinprice.xml

```diff
@@ -45,16 +45,16 @@
   <Optimizers>
     <SPSA name="opter">
       <initialization>
-        <limit>2000</limit>
+        <limit>4000</limit>
```

Comment:
Why do you increase the limit? Should the convergence not be faster with the new gradient?